webapp/src/routes/Profile/ProfilePageSponsor.js

```diff
@@ -621,7 +614,7 @@ const ProfilePageSponsor = ({ profile }) => {
           <Box className={classes.rowBoxLeft}>
             <Typography className={classes.rowTitle} variant="subtitle1">{t('profile.images')}</Typography>
             <Box>
-              <img className={classes.img} src={images[activeStep]} />
+              <img className={classes.img} src={images[activeStep]} alt='Profile sponsor' />
```

Comment:
Maybe we can use alt in english and spanish

Reply:
You right, working on